OpenEdXMobile/src/main/java/org/edx/mobile/player/TranscriptManager.java

```diff
@@ -55,8 +55,11 @@ public boolean has(String url) {
      * @throws IOException
      */
     public void put(String url, String response) throws IOException {
+        final File transcriptDir = getTranscriptDir();
+        if (transcriptDir == null) throw new IOException("Transcript directory doesn't exist");
```

Comment:
I suggest changing "doesn't exist" to "not found" or "nor available" in all these methods.